pkg/config/config_template.yaml

```diff
@@ -1735,6 +1735,7 @@ api_key:
 ## @param dogstatsd_stats_enable - boolean - optional - default: false
 ## @env DD_DOGSTATSD_STATS_ENABLE - boolean - optional - default: false
 ## Publish DogStatsD's internal stats as Go expvars.
+## Enabling this will write DogstatsD metrics that agent received to log files
```

Comment:
Is there a way to disable this log file, but keep stats enabled?

It may be better to have a separate option for this feature.

Reply:
This could be a good motivation for adding a separate config `dogstatsd_log_file_max_size`, have it default to the same as the agent log and if its set to zero that would disable the `dogstatsd_stats` logging. What do you think?

Reply:
So just checking the logic here, but if that is the case, i think it makes more sense to just change the env var name all together to something more like:
```
dogstatsd_stats_logging: 0 or 1 
```
with 0 meaning disabled dogstatsd stats logging
and 1 for using the agent's `log_file_max_rolls` to determine how many log file will be rolled over. 

Is that correct?

Reply:
+ I added the suggested env var so now there are 3 conditions for a log to be generated on the log file:
1 and 2 being these 2 env vars:
```
dogstatsd_metrics_stats_enable: true
dogstatsd_stats_logging: false 
```
and 3rd condition being a null check for the logger.